src/UnivPoly.jl

```diff
@@ -4,6 +4,14 @@
 #
 ###############################################################################
 
+function content(a::UniversalPolyRingElem)
+   z = zero(base_ring(a))
+   for c in coefficients(a)
+      z = gcd!(z, c)
+   end
+   return z
```

Comment:
Much better to delegate to the wrapped wring, which may have an optimized implementation, e.g. `content(::QQMPolyRing)` runs in constant time...
```suggestion
   return content(data(a))
```